### PR TITLE
ADEN-5214 | Veles - allow bidding up to $50

### PR DESCRIPTION
**Pull request by kvas-damian**
https://wikia-inc.atlassian.net/browse/ADEN-5214

ping @Wikia/adeng

### CODE REVIEW

File: extensions/wikia/AdEngine/js/spec/lookup/prebid/adaptersPricesTracker.spec.js [hunk 1]
```diff
@@ -8,7 +8,7 @@ describe('ext.wikia.adEngine.lookup.prebid.adaptersPricesTracker', function () {
 	function getModule() {
 		return modules['ext.wikia.adEngine.lookup.prebid.adaptersPricesTracker'](
 			mocks.adaptersRegistry,
```
Comment (WikiaTech) [inline extensions/wikia/AdEngine/js/spec/lookup/prebid/adaptersPricesTracker.spec.js:10]: MAJOR Move the declaration of "mocks" before this usage. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AVariableDeclarationAfterUsage)

File: extensions/wikia/AdEngine/js/lookup/prebid/adaptersPerformanceTracker.js [hunk 2]
```diff
@@ -1,10 +1,10 @@
 define('ext.wikia.adEngine.lookup.prebid.adaptersPerformanceTracker', [
 	'ext.wikia.adEngine.adTracker',
 	'ext.wikia.adEngine.lookup.prebid.adaptersRegistry',
-	'ext.wikia.adEngine.lookup.prebid.priceGranularityHelper',
+	'ext.wikia.adEngine.lookup.prebid.bidHelper',
 	'ext.wikia.adEngine.utils.timeBuckets',
 	'ext.wikia.adEngine.wrappers.prebid'
-], function (adTracker, adaptersRegistry, priceGranularityHelper, timeBuckets, prebid) {
+], function (adTracker, adaptersRegistry, bidHelper, timeBuckets, prebid) {
```
Comment (WikiaTech) [inline extensions/wikia/AdEngine/js/lookup/prebid/adaptersPerformanceTracker.js:7]: MAJOR Function has 5 parameters which is greater than 4 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)

File: extensions/wikia/AdEngine/js/spec/lookup/prebid/bidHelper.spec.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
+/*global beforeEach, describe, it, modules, expect, spyOn*/
+describe('ext.wikia.adEngine.lookup.prebid.bidHelper', function () {
+	'use strict';
+
+	function getModule() {
+		return modules['ext.wikia.adEngine.lookup.prebid.bidHelper'](
+			mocks.bidHelper
```
Comment (WikiaTech) [inline extensions/wikia/AdEngine/js/spec/lookup/prebid/bidHelper.spec.js]: MAJOR Move the declaration of "mocks" before this usage. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AVariableDeclarationAfterUsage)

**Comment by WikiaTech** [comment]
SonarQube analysis reported 3 issues
* ![MAJOR][MAJOR] 3 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Comment by WikiaTech** [comment]
SonarQube analysis reported 2 issues
* ![MAJOR][MAJOR] 2 major

Watch the comments in this conversation to review them.

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Review by fraszczakszymon — APPROVED** [review]

**Comment by WikiaTech** [comment]
SonarQube analysis reported 6 issues
* ![MAJOR][MAJOR] 6 major

Watch the comments in this conversation to review them.

#### 4 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MAJOR][MAJOR] [playerTracker.js#L13](https://github.com/Wikia/app/blob/70846bfae8ac592cd79244ade688e18089536243/extensions/wikia/AdEngine/js/video/player/playerTracker.js#L13): Function has 10 parameters which is greater than 4 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)
1. ![MAJOR][MAJOR] [playerTracker.js#L38](https://github.com/Wikia/app/blob/70846bfae8ac592cd79244ade688e18089536243/extensions/wikia/AdEngine/js/video/player/playerTracker.js#L38): Function has a complexity of 19 which is greater than 10 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AFunctionComplexity)
1. ![MAJOR][MAJOR] [playerTracker.js#L38](https://github.com/Wikia/app/blob/70846bfae8ac592cd79244ade688e18089536243/extensions/wikia/AdEngine/js/video/player/playerTracker.js#L38): Function has 5 parameters which is greater than 4 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)
1. ![MAJOR][MAJOR] [playerTracker.js#L90](https://github.com/Wikia/app/blob/70846bfae8ac592cd79244ade688e18089536243/extensions/wikia/AdEngine/js/video/player/playerTracker.js#L90): Function has 5 parameters which is greater than 4 authorized. [rule](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AExcessiveParameterList)

[MAJOR]: https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR'

**Pull request merged**